backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlInfo.java

```diff
@@ -47,8 +47,15 @@
 	 */
 	private final boolean translated;
 
+	private final boolean translations;
+
 	private final String tableName;
 	private final String keyColumnName;
+	private final String adLanguage;
+
+	@Default
+	@NonNull
+	private final ImmutableList<String> keyColumnNames = ImmutableList.of();
```

Comment:
you already have a `String keyColumnName`. Why keyColumnNames is needed?
Asking because translation is not supported by composed primary key tables.

Reply:
because i use `POTrlInfo` for translation tables now as well in `de.metas.i18n.po.POTrlRepository#updateBaseTableOnTranslationChange()` and _trl tables have 2 keycolumns

Reply:
in case of translatable tables, the base table always have a single primary key
and _Trl table always have a composed key of (parent link and AD_Language).